sections/clause_07b_sensing_entities.adoc

```diff
@@ -32,25 +45,592 @@ requirement:: {identifier}/req/datamodel/sensing/thing/properties
 requirement:: {identifier}/req/datamodel/sensing/thing/relations
 ====
 
-The Thing is a cool Entity that can be anything.
-Say something about OMS-Host, OMS-Deployment, FeatureOfInterest and Locations
+The SensorThings API follows the ITU-T definition, i.e., with regard to the Internet of Things, a thing is an object of the physical world (physical things) or the information world (virtual things) that is capable of being identified and integrated into communication networks [ITU-T Y.2060]. A Thing is related to the Platform entity as described in Section 4.9.2.1 of [OGC 16-079] in a way that any entity that can be modelled as a Thing MAY be subsequently translated to a Platform and vice versa.
```

Comment:
In 18-088, [Section 7.4](https://docs.ogc.org/is/18-088/18-088.html#observations-measurements) describes the OMS links. I thought to keep it that way instead of discussing it along with every entity?

Reply:
Good point, we have to update that section too.